src/background/Wallet/Wallet.ts

```diff
@@ -1182,7 +1184,7 @@ export class Wallet {
     return result;
   }
 
-  async resetEthereumChain({
+  async removeEthereumChainConfig({
```

Comment:
I agree that these method names are confusing, but not sure that this is the best name. We still call `chainConfigStore.removeEthereumChain` inside

Reply:
Right, after taking another look, I can see that renaming didn't improve readability...

Reply:
✅  Reverted